src/iso7816.h

```diff
@@ -27,14 +27,13 @@ struct iso7816_header {
 	uint8_t lc3;
 })
```

Comment:
Note that, by similar logic, this also doesn't actually need to be packed, but doing so is a harmless no-op and reduces churn